packages/website/src/components/TableOfContentItems.tsx

```diff
@@ -48,7 +48,7 @@ export function TableOfContentItems({
 					</Group>
 				</Box>
 			)),
-		[properties, colorScheme],
```

Comment:
I don't think this change is correct, yes the eslint warning says something different, but 🤔